src/scripts/dungeons/DungeonRunner.ts

```diff
@@ -222,10 +222,11 @@ class DungeonRunner {
     public static dungeonWon() {
         if (!DungeonRunner.dungeonFinished()) {
             DungeonRunner.dungeonFinished(true);
+            if (!App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]()) {
+                DungeonRunner.dungeon.rewardFunction();
+            }
             GameHelper.incrementObservable(App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]);
             MapHelper.moveToTown(DungeonRunner.dungeon.name);
-            DungeonRunner.dungeon.rewardFunction();
-            // TODO award loot with a special screen
```

Comment:
Maybe keep the TODO? I can see us maybe wanting to still do that at some point.

Reply:
Then it should be an issue imo. The code is not the correct place for suggestions :)